ssh/DOCS.md

```diff
@@ -65,14 +72,22 @@ Specifies whether TCP forwarding is permitted or not.
 
 **Note**: _Enabling this option lowers the security of your SSH server! Nevertheless, this warning is debatable._
 
+
 ## Network
-To enable ssh access via the network, you need to enter the port number ‘22’ or the port you want to use. This will map that port from the hassio host into the running “Terminal & SSH” container.
+
+To enable ssh access via the network, you need to specify which external port to use. 
```

Comment:
The term `SSH` needs to be capitalized, the external port might be confusing for the end user.

Reply:
Fixed ssh, and reworded "external port" to try to avoid the confusion, thanks.